attribute-service-impl/src/main/java/org/hypertrace/core/attribute/service/model/AttributeMetadataModel.java

```diff
@@ -366,7 +357,8 @@ public boolean equals(Object o) {
         && Objects.equals(supportedAggregations, that.supportedAggregations)
         && Objects.equals(sources, that.sources)
         && Objects.equals(metadata, that.metadata)
-        && Objects.equals(definition, that.definition);
+        && Objects.equals(definition, that.definition)
```

Comment:
There's half codecov for this. It's yellow instead of green

Reply:
I'd guess that's because this is a ton of branches, and we're not writing tests for each permutation of model equality so it never gets to this bottom branch. Do you think it's worth writing tests for them all considering this method is generated? Also from a quick peak around, I don't believe equal/hashcode are used on this model anyway.

Reply:
ok. I always shoot for 100% but i guess it's not crucial in this case.